main.js

```diff
@@ -0,0 +1,4 @@
+var tableElement = document.getElementById('table-el');
```

Comment:
You are saving your table into the selector, but the table body is probably what you want to select, since that is where you will be appending your data.